src/Build.UnitTests/Evaluation/Evaluator_Tests.cs

```diff
@@ -4244,6 +4244,7 @@ public void VerifyDTDProcessingIsDisabled()
         /// If DTD processing is disabled, the server should not receive any connection request.
         /// </summary>
         [Fact]
+        [SkipOnPlatform(TestPlatforms.Windows, "This test is flaky. We should try to fix it if possible. https://github.com/dotnet/msbuild/issues/7623")]
```

Comment:
I prefer the dedicated-to-purpose attribute

```suggestion
        [ActiveIssue("https://github.com/dotnet/msbuild/issues/7623")]
```